test/test-queue-that.js

```diff
@@ -310,6 +310,65 @@ describe('createQueueThat', function () {
       expect(options.process.callCount).to.be(3)
     })
 
+    it('should report repeated items on process error', function () {
+      localStorageAdapter.setQueue(_.range(4))
```

Comment:
Indeed, the server doesn't really care about the content so it makes sense. Not terribly good for more general use cases but I don't they exist right now.